applications/RomApplication/custom_utilities/rom_auxiliary_utilities.cpp

```diff
@@ -335,6 +335,50 @@ std::vector<IndexType> RomAuxiliaryUtilities::GetHRomConditionParentsIds(
     return parent_ids;
 }
 
+std::vector<IndexType> RomAuxiliaryUtilities::AddModelPartElementsToHROM(
+    const ModelPart& rModelPart,
+    const ModelPart& rModelPartWithElementsToInclude,
+    std::map<std::string, std::map<IndexType, double>>& rHRomWeights)
+{
+    std::vector<IndexType> new_element_ids;
+    auto& r_elem_weights = rHRomWeights["Elements"];
+
+    for (const auto& r_elem : rModelPartWithElementsToInclude.Elements()) {
+        IndexType element_id = r_elem.Id();
+        
+        // Check if the element is already added
+        if (r_elem_weights.find(element_id - 1) == r_elem_weights.end()) {
+            // If it's not added, we add it to the HROM weights and the new_element_ids list
+            r_elem_weights[element_id - 1] = 0; // Adjust the value 0 based on your needs.
```

Comment:
When would it make sense to add manually a different value than zero? An approximated reduced system of equation could be achived only if the rest of the weights were also modified, isnt it?